### ISSUE TITLE
Error Unsupport platform linux_x86_64

### ISSUE THREAD

**Issue by KomarovAlea** [dashingsoft/pyarmor#73]
https://github.com/dashingsoft/pyarmor/blob/1762ce3de69738bf8025a39b1a121bd73de8b9d8/src/utils.py#L57

1. I get error when I clone project, make setup and start: 
`python pyarmor -v` 
on platform **linux_x86_64**

because in file config.py you separate support_platform in two lists and get always second list of platform.
```
support_platforms = [
    (
        ('win32', 'win32'),
        ('win_amd64', 'win_amd64'),
        ('manylinux1_i686', 'linux_i386'),
        ('manylinux1_x86_64', 'linux_x86_64'),
        ('macosx_10_11_x86_64', 'macosx_x86_64'),
    ),
    (
        ('linux_ppc64', 'ppc64le'),
        ('linux_armv5', 'armv5'),
        ('linux_armv7', 'armv7'),
        ('linux_aarch32', 'armv8.32-bit'),
        ('linux_aarch64', 'armv8.64-bit'),
        ('linux.musl_x86_64', 'alpine'),
        ('ios_arm64', 'ios.arm64'),
        ('freebsd_x86_64', 'freebsd'),
    )
]
```
then I make monkeypatch:
`if plat_name not in dict(support_platforms[0]):`
2. but got new error:
my plat_name is '**linux_x86_64**' but you make dictionary from support_platforms and my platform have key 'manylinux1_x86_64'.
So I got error: Unsupport platform linux_x86_64 because python3 cant find 'linux_x86_64' in keys 'manylinux1_i686', 'manylinux1_x86_64', 'macosx_10_11_x86_64'...
I make second monkeypatch in config.py change platform key:

support_platforms = [
    (
        ('win32', 'win32'),
        ('win_amd64', 'win_amd64'),
        ('manylinux1_i686', 'linux_i386'),
        ('**linux_x86_64**', 'linux_x86_64'),
        ('macosx_10_11_x86_64', 'macosx_x86_64'),
 ...
and this will work for my.

**Comment by jondy** [comment]
Thank you very much. Because I always test pyarmor by pip installed package, :). 
Maybe it's better to fix it by this way, change the check statement as :
```
if plat_name not in dict(support_platforms[0]).values():
```

**Comment by jondy** [comment]
Fix it by this commit 74cba64
